docker-compose.override.yml

```diff
@@ -5,7 +5,7 @@ services:
   # FRONTEND #############################################
   ########################################################
   frontend:
-    image: gradido/frontend:development
+    # image: gradido/frontend:development

```

Comment:
Comment out this and all the others and leave them in the code?
Or remove them?

Reply:
Renamed